sys/net/gnrc/netif/Kconfig

```
@@ -21,7 +21,9 @@ config GNRC_NETIF_MSG_QUEUE_SIZE_EXP
         the queue.
 
 config GNRC_NETIF_IPV6_ADDRS_NUMOF
-    int "Maximum number of unicast and anycast addresses per interface"
+    int
+    prompt "Maximum number of unicast and anycast addresses per interface"
```

I think this change has the same effect as the original state

```suggestion
    int "Maximum number of unicast and anycast addresses per interface"
```